parts/k8s/addons/kubernetesmasteraddons-keyvault-flexvolume-installer.yaml

```diff
@@ -47,3 +47,87 @@ spec:
         name: volplugins
       nodeSelector:
         beta.kubernetes.io/os: linux
+---
+apiVersion: extensions/v1beta1
```

Comment:
@ritazh Does it make sense to implement these as add'l DaemonSets in the same addon? (does all or nothing install make sense functionally)

Reply:
blobfuse flexvolume and smb flexvolume are two different drivers. They should be installed as separate addons. 

@andyzhangx FYI

Reply:
@jackfrancis agree with @ritazh, we should use seperate daemonset